documentation/src/docs/dev-guide.md

```diff
@@ -66,11 +71,11 @@ You can follow [here](https://ci.apache.org/projects/flink/flink-docs-stable/dev
 
 Add the below snippet to dependencies section of build.gradle in the app directory, connector dependencies should be part of the shadow jar. For flink connector dependency, we need to choose the connector which aligns the Flink major version and Scala version if you use Scala, along with the same Pravega version you run.
 
-```groovy
-compile group 'org.apache.flink', name: 'flink-streaming-java_2.12', version: '1.11.2'
+<pre><code {...{ "className": "language-groovy" }}>
+{`compile group 'org.apache.flink', name: 'flink-streaming-java_${versions.scala}', version: '${versions.flink}'
 
-flinkShadowJar group: 'io.pravega', name: 'pravega-connectors-flink-1.11_2.12', version: '0.9.0'
-```
+flinkShadowJar group: 'io.pravega', name: 'pravega-connectors-flink-${versions['flink-minor']}_${versions.scala}', version: '${versions['flink-connectors']}'`}
+</code></pre>
```

Comment:
What are the correct latest versions here, @crazyzhou? Version info comes from here: https://github.com/pravega/docs/commit/cf71c4b29058a16ff897c3a14cbccc4577d62e0d#diff-21a0578522c567f65881e4e8dc3aaace32d60dbdabf2730082b7d9da9f6cb4d0R13

Reply:
```
flink: 1.13.2
flink.minor: 1.13
scala: 2.12
flink-connectors: 0.10.1
```